yaml/restaurant_item.yml

```diff
@@ -0,0 +1,48 @@
+RestaurantItemEndpoint
```

Comment:
![image](https://user-images.githubusercontent.com/6318579/105361600-ecf77200-5c0a-11eb-9710-2ecd10495f28.png)

1. please set the name of section from `default` to some different, like `restaraunts`. big APIs have several sections e.g. `restaraunts`, `users`, `orders`
2. check Flasgger docs and find how to set Swagger page name and description, e.g. not `powered by Flasgger` but `Delivery System API`

Reply:
fixed